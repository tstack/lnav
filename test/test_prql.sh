#! /bin/bash

export TZ=UTC
export YES_COLOR=1
unset XDG_CONFIG_HOME

run_cap_test ${lnav_test} -n \
    -c ";from access_log | take 1" \
    ${test_dir}/logfile_access_log.0

run_cap_test ${lnav_test} -n \
    -c ";from access_log | take abc" \
    ${test_dir}/logfile_access_log.0

run_cap_test ${lnav_test} -n \
    -c ";from bro_http_log | stats.hist bro_host slice:'1m'" \
    ${test_dir}/logfile_bro_http.log.0
