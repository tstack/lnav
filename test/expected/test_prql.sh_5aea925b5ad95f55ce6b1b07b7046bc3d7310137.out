[1m[4m        tslice         [0m[1m[4m [0m[1m[4m                                                           v                                                            [0m[1m[4m [0m
2011-11-03 00:19:00.000 [7m{"a.fsdn.com":33,"Other":31,"ad.[0m[7mdoubleclick.net":10,"www.gooâ‹¯[0m[7m"\xaf"dia.com"[0m[7m:6,"s0.2m[0m[7mdn.net":[0m[7m5,"googl[0m[7meads.g.[0m[7mdoubl[0m[7mecli[0m[7mck[0m.net":3} 
2011-11-03 00:20:00.000 [7m{"www.bro-ids.org":24,"[0m[7mwww.google.com[0m[7m":15,"[0m[7msearc[0m[7mh.twi[0m[7mtter[0m[7m.[0m[7mc[0m[7mâ‹¯[0m"\x8b""\xaf"ds.org":2,"ad.doubleclick.net":1,"cont-sjl-1.pandora.com":1} 
[1m2011-11-03 00:21:00.000[0m[1m [0m[1m[7m{[0m[1m[7m"[0m[1m[7mw[0m[1m[7mw[0m[1m[7mw[0m[1m.google.com":2,"ajax.googleapis.com":1,"api.twitter.com":1,"search.twitter.com":1,"www.bro-ids.org":1}              [0m
